src/main.js

```diff
@@ -134,14 +181,25 @@ export class Draggable {
                 unbind(this.document, "pointercancel", this._pointerup);
                 unbind(this.document, "contextmenu", preventDefault);
 
+                if (this.clickMoveClick){
+                    unbind(this.document, "keydown", this._keydown);
+                }
+
                 this._releaseHandler(e);
             }
         };
+
+        this._keydown = (e) => {
+            if (e.keyCode === ESC) {
+                this.cancelDrag(e);
+                this._cancelHandler(e);
+            }
```

Comment:
It seems like this behavior can be an option - `navigable: true`. 

We discussed this offline - some suites like Kendo Angular/React might want to have their own events or specific navigation and want to cancel the drag on Escape optionally.